### PR TITLE
Feature/conda

### PR DESCRIPTION
**Pull request by henrikstranneheim**

### CODE REVIEW

File: conda/conda_conventions.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Conda conventions at Clinical Genomics
+
+- Avoid installing anything in the conda default environment (base) if possible
+- Naming conventions for conda environments: `[env_type]_[creation_date]_[logical_name]_[signature]`
```
Comment (sylvinite) [inline conda/conda_conventions.md]: Personally I would prefer <software name>_<P/D><version/timestamp> and signature only in the case of devel environments. Trims down the clutter and keeps the conda name just as informative.

Comment (henrikstranneheim) [reply conda/conda_conventions.md]: Also shorter which is nice

**Comment by b4ckm4n** [comment]
Looks cool 👍

**Comment by henrikstranneheim** [comment]
I have updated the conda conventions. I will merge this PR at the end of the day, so anyone who wants to influence this right now - should start reviewing. 

Let's not leave these PR hanging around. They are small and easy to review - aim to do it within 24 hours.

**Comment by ingkebil** [comment]
Long conda names don't natter so much as long as we can auto complete them easily. Which we can:

https://conda.io/docs/user-guide/configuration/enable-tab-completion.html

**Comment by henrikstranneheim** [comment]
I think that is only for conda and not for "source activate [env]"

**Comment by ingkebil** [comment]
Well that sucks. Doesn't seem we can auto complete env names. Then typing a 30+ char env name is really annoying. :(

**Comment by henrikstranneheim** [comment]
Well I do not think it is that bad:
Today: prod170926 (10 char)
Tomorrow: P_cg_170926 (11 char)

But a lot more control and readability

**Comment by ingkebil** [comment]
Aha, sorry! didn't read the suggested name in documentation

**Comment by henrikstranneheim** [comment]
Alright! I will merge this and we can have it as a point on the next production meeting

**Pull request merged**